### ISSUE TITLE
Investigate the discrepancy in default hyperparams compared to LightGBM

### ISSUE THREAD

**Issue by ogrisel** [ogrisel/pygbm#32]
Possible culprits:

- `shrinkage` / `learning_rate`
- `min_samples_leaf`
- `min_child_samples`

See details in https://github.com/ogrisel/pygbm/issues/30#issuecomment-435091127.

**Comment by guolinke** [comment]
@NicolasHug I think you used the wrong parameter name in that code.

**Comment by ogrisel** [comment]
Indeed. It's actually the pygbm handling of `min_samples_leaf` that is broken. See: #34.
